components/status-message/assets/StatusMessage.scss

```diff
@@ -0,0 +1,54 @@
+@import "@hods/sass-base";
+@import "govuk-frontend/govuk/components/button";
+
+$govuk-button-colour: govuk-colour("white");
+$govuk-button-text-colour: govuk-colour("black");
+
+.hods-status-message {
+  color: govuk-colour("white");
+  background-color: govuk-colour("blue");
+  @include govuk-responsive-padding(4, "all");
+
+  &__extra {
+    font-weight: normal;
```

Comment:
Is this necessary? `font-weight` is normal by default

Reply:
Yes. It would inherit the bold of the parent element otherwise.

Reply:
It doesn't for me unless I also remove line 41. It stays the same weight when I delete lines 12-14